packages/plugin-kit/package.json

```diff
@@ -32,21 +32,25 @@
   "homepage": "https://github.com/eslint/rewrite#readme",
   "scripts": {
     "build:dedupe-types": "node ../../tools/dedupe-types.js dist/cjs/index.cjs dist/esm/index.js",
-    "build:cts": "node -e \"fs.copyFileSync('dist/esm/index.d.ts', 'dist/cjs/index.d.cts')\"",
+    "build:cts": "node -e \"fs.writeFileSync('dist/cjs/index.d.cts', fs.readFileSync('dist/esm/index.d.ts', 'utf-8').replaceAll('import(\\\"./types.ts\\\")', 'import(\\\"./types.ts\\\", { with: { \\\"resolution-mode\\\": \\\"import\\\" } })'))\"",
```

Comment:
This is now long enough that I think we should pull it out into a separate script file so we can appropriately document and lint it.

Reply:
Done, thanks!